### PR TITLE
Updates in Mongodb atlas, Tracing and fedramp docs

### PR DESCRIPTION
**Pull request by himanshu219**
## Purpose of this pull request

This pull request
* updates fedramp capabilities section.
* updates mongodb atlas collection docs.
* updates v2 app install / upgrade / uninstall instructions for tracing/rum apps.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/databases/mongodb-atlas.md [hunk 1]
```diff
@@ -323,18 +321,21 @@ In this section, you deploy the SAM application, which creates the necessary res
 
 The lambda function should be working now in sending logs to Sumo. You can check the CloudWatch logs in **Monitor** > **Logs** to see the logs of the function.
 
-#### Step 3B: Configure Collection for Multiple Projects (optional)
+13. Configuring collection for multiple projects (assuming you are already collecting Atlas data for one project). This task requires that you do the following:
```
Comment (jpipkin1) [inline docs/integrations/databases/mongodb-atlas.md:324]: @himanshu219, this should be a new subsection. For example:

#### Configure collection for multiple projects

If you are already collecting Atlas data for one project, perform the following steps to configure for additional projects.

Comment (himanshu219) [reply docs/integrations/databases/mongodb-atlas.md:324]: sure let me create a separate subsection

File: docs/integrations/databases/mongodb-atlas.md [hunk 2]
```diff
@@ -387,88 +388,24 @@ This task makes the following assumptions:
      */5 * * * *  /usr/bin/python3 -m sumomongodbatlascollector.main > /dev/null 2>&1
      ```
 5. Configuring collection for multiple projects (assuming you are already collecting Atlas data for one project). This task requires that you do the following:
```
Comment (jpipkin1) [inline docs/integrations/databases/mongodb-atlas.md:390]: @himanshu219, you should create a new section here too.

**Review by jpipkin1 — APPROVED** [review]
Approved, @himanshu219. I made some minor changes to the MongoDB Atlas article, mostly to list numbering and screenshot tagging (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4752/commits/140ab941d81329f6efb7fc60d6fc97d594a53f5a)").

**Comment by himanshu219** [comment]
Thanks @jpipkin1  the border and fixed width to images looks nice! should we follow the same standard for future docs?

**Pull request merged**